app/src/interfaces/select-icon/select-icon.vue

```diff
@@ -103,19 +143,35 @@ function onKeydownInput(e: KeyboardEvent, activate: () => void) {
 		</template>
 
 		<div class="content" :class="width">
-			<template v-for="(group, index) in filteredIcons" :key="group.name">
-				<div v-if="group.icons.length > 0" class="icons">
-					<v-icon
-						v-for="icon in group.icons"
-						:key="icon"
-						:name="icon"
-						:class="{ active: icon === value }"
-						clickable
-						@click="setIcon(icon)"
-					/>
-				</div>
-				<v-divider v-if="group.icons.length > 0 && index !== filteredIcons.length - 1" />
-			</template>
+			<DynamicScroller
+				:items="virtualRows"
+				:min-item-size="32"
+				:buffer="400"
+				:prerender="10"
+				:size-field="'32'"
+				key-field="rowIndex"
+				page-mode
+			>
+				<template #default="{ item }">
+					<DynamicScrollerItem :item="item" active>
+						<div v-if="item.type === 'header'" class="group-header">
+							<div class="group-name">
+								{{ item.groupName }}
+							</div>
+						</div>
```

Comment:
Let’s use an existing component like `<v-divider>` for the headers. Also, set `--v-divider-label-color: var(--theme--foreground-subdued)` for that `<v-divider>` and update `.group-header` (CSS) accordingly.

```suggestion
						<v-divider v-if="item.type === 'header'" inline-title>
							{{ item.groupName }}
						</v-divider>
```

Reply:
Would look like in the image below. However, …

- [ ] Let’s align the icons to the left (inline-start).
- [ ] Additionally, when opening the menu, let’s calculate the `iconsPerRow` so that we use the entire space. Alternatively, we could display the window with a smaller width.

<img width="1722" height="1028" alt="v-divider" src="https://github.com/user-attachments/assets/3f183295-5e09-4358-946b-e43fc432e0d4" />

Reply:
Here is what I have now. Sometimes there will be a space on the right side, depending on how many 24px icons can fit in the available space, but I think this works well.